### ISSUE TITLE
Isotope module breaks cantera simulation

### ISSUE THREAD

**Issue by KEHANG** [ReactionMechanismGenerator/RMG-Py#796]
As RMG-tests reveals, the newly added isotope module produces unconventional chemkin file (with isotopes in element list) shown in below format:
```
ELEMENTS
	H
	D /2.014/
	T /3.016/
	C
	CI /13.003/
	O
	OI /18.000/
	N
	Ne
	Ar
	He
	Si
	S
	Cl
END
```

, which can not be recognized by cantera `ck2cti` tool, and resulting cti file will be like
```
elements="H D /2.014/ T /3.016/ C Ci /13.003/ O Oi /18.000/ N Ne Ar He Si S Cl",
```
leading to simulation error:
```
***********************************************************************
CanteraError thrown by addElementsFromXML:
no data for element /2.014/
***********************************************************************
```

In addition to fixing this issue, we should also add some unit test to give out warning of this kind of change, also code reviewer for pull request need to look at RMG-tests as well before merging.

**Comment by KEHANG** [comment]
cantera's recent issue made by Nick (https://github.com/Cantera/cantera/commit/7d18120c4d4fb5d669611e69fa2c30be5c4cb9b2) is exactly same as this issue. And cantera has already made changes to cooperate our chemkin isotope format, feel odd the problem is still there.

**Comment by rwest** [comment]
Update the cantera build? The fix is quite recent.

**Comment by rwest** [comment]
Not sure I heard this correctly on the call, but if it's true that our CHEMKIN files now include isotopes even if we're not using or requesting them (I.e. We're not using the new isotope module) then I think this should be changed. I.e. Out of the box, RMG should make CHEMKIN files without extra isotope stuff.

**Comment by alongd** [comment]
I sometimes get Cantera crush even for known elements, if these elements aren't used in any species (#748). I would recommend having only the 'active' elements in each RMG output.

**Comment by KEHANG** [comment]
@alongd this is related issue, but isotope mechanism still can not be fixed by removing unnecessary elements. Maybe create another issue? For your issue, I kind of experienced before, it seems to happen when you have too few reactions (fewer than element numbers?)

**Comment by KEHANG** [comment]
isotope mechanism conversion to cti fixed by PR #801.
